bin/boot_docker

```diff
@@ -138,10 +138,10 @@ docker build -t single_cell_docker:$DOCKER_IMAGE_VERSION -f Dockerfile .
 if [[ $PASSENGER_APP_ENV = "production" ]]
 then
 	# generate random secret key for secure cookies
-	docker run -d --name $CONTAINER_NAME -p 80:80 -p 443:443 -h $PROD_HOSTNAME -v $PROJECT_DIR:/home/app/webapp:rw -v $PROJECT_DIR/data:/home/app/webapp/data:rw -v $PROJECT_DIR/log/nginx:/var/log/nginx:rw -e PASSENGER_APP_ENV=$PASSENGER_APP_ENV -e MONGO_LOCALHOST=$MONGO_LOCALHOST -e PROD_DATABASE_PASSWORD=$PROD_DATABASE_PASSWORD -e SERVICE_ACCOUNT_KEY=$SERVICE_ACCOUNT_KEY -e READ_ONLY_SERVICE_ACCOUNT_KEY=$READ_ONLY_SERVICE_ACCOUNT_KEY -e PORTAL_NAMESPACE=$PORTAL_NAMESPACE -e SECRET_KEY_BASE=$SECRET_KEY_BASE -e PROD_HOSTNAME=$PROD_HOSTNAME -e SENDGRID_USERNAME=$SENDGRID_USERNAME -e SENDGRID_PASSWORD=$SENDGRID_PASSWORD -e OAUTH_CLIENT_ID=$OAUTH_CLIENT_ID -e OAUTH_CLIENT_SECRET=$OAUTH_CLIENT_SECRET -e TCELL_AGENT_APP_ID=$TCELL_AGENT_APP_ID -e TCELL_AGENT_API_KEY=$TCELL_AGENT_API_KEY -e SENTRY_DSN=$SENTRY_DSN -e GA_TRACKING_ID=$GA_TRACKING_ID single_cell_docker:$DOCKER_IMAGE_VERSION
+	docker run -d --name "$CONTAINER_NAME" -p 80:80 -p 443:443 -h "$PROD_HOSTNAME" -v "$PROJECT_DIR:/home/app/webapp:rw" -v "$PROJECT_DIR/data:/home/app/webapp/data:rw" -v "$PROJECT_DIR/log/nginx:/var/log/nginx:rw" -e PASSENGER_APP_ENV="$PASSENGER_APP_ENV" -e MONGO_LOCALHOST="$MONGO_LOCALHOST" -e PROD_DATABASE_PASSWORD="$PROD_DATABASE_PASSWORD" -e SERVICE_ACCOUNT_KEY="$SERVICE_ACCOUNT_KEY" -e READ_ONLY_SERVICE_ACCOUNT_KEY="$READ_ONLY_SERVICE_ACCOUNT_KEY" -e PORTAL_NAMESPACE="$PORTAL_NAMESPACE" -e SECRET_KEY_BASE="$SECRET_KEY_BASE" -e PROD_HOSTNAME="$PROD_HOSTNAME" -e SENDGRID_USERNAME="$SENDGRID_USERNAME" -e SENDGRID_PASSWORD="$SENDGRID_PASSWORD" -e OAUTH_CLIENT_ID="$OAUTH_CLIENT_ID" -e OAUTH_CLIENT_SECRET="$OAUTH_CLIENT_SECRET" -e TCELL_AGENT_APP_ID="$TCELL_AGENT_APP_ID" -e TCELL_AGENT_API_KEY="$TCELL_AGENT_API_KEY" -e SENTRY_DSN="$SENTRY_DSN" -e GA_TRACKING_ID="$GA_TRACKING_ID" single_cell_docker:$DOCKER_IMAGE_VERSION
```

Comment:
This line is over 900 characters long, making it hard to discern what changed.  

Line breaks (as done e.g. [here](https://github.com/broadinstitute/single_cell_portal_core/pull/282/files#diff-8808495d9da0b880b99ea0bb7254142cR70)) would help readability.

Reply:
Yeah, that should have been done a long time ago.  I'll take care of that.